app/src/ui/changes/commit-message.tsx

```diff
@@ -229,7 +229,7 @@ export class CommitMessage extends React.Component<
   }
 
   private onSubmit = () => {
-    this.createCommit()
+    this.showChanges()
```

Comment:
i think this one was supposed to stay as createCommit since it's the submit button to create the commit (in fairness it's confusing)

Reply:
😅 

...

Fixed in f5a10ca 🙇